template/src/utilities/networking.js

```diff
@@ -0,0 +1,102 @@
+import instance from "./axiosInstance";
+
+export function setHeader (property, data) {
+  instance.defaults.headers.common[property] = data;
+};
+
+export function setBaseURL (baseURL) {
+  instance.defaults.baseURL = baseURL;
+};
+
+export function setToken (token, type = null) {
```

Comment:
setAuthHeader

Reply:
I think setToken name is the most obvious name. This name shouldn't be changed.